FunctionSignatureMap.php

```diff
@@ -6656,16 +6655,16 @@
 'maxdb_warning_count' => ['int', 'link'=>'resource'],
 'mb_check_encoding' => ['bool', 'var='=>'string', 'encoding='=>'string'],
 'mb_chr' => ['string|false', 'cp'=>'int', 'encoding='=>'string'],
-'mb_convert_case' => ['string', 'sourcestring'=>'string', 'mode'=>'int', 'encoding='=>'string'],
-'mb_convert_encoding' => ['string', 'str'=>'string', 'to_encoding'=>'string', 'from_encoding='=>'mixed'],
-'mb_convert_kana' => ['string', 'str'=>'string', 'option='=>'string', 'encoding='=>'string'],
+'mb_convert_case' => ['string|false', 'sourcestring'=>'string', 'mode'=>'int', 'encoding='=>'string'],
+'mb_convert_encoding' => ['string|false', 'str'=>'string', 'to_encoding'=>'string', 'from_encoding='=>'mixed'],
+'mb_convert_kana' => ['string|false', 'str'=>'string', 'option='=>'string', 'encoding='=>'string'],
 'mb_convert_variables' => ['string|false', 'to_encoding'=>'string', 'from_encoding'=>'array|string', '&rw_vars'=>'string|array|object', '&...rw_vars='=>'string|array|object'],
 'mb_decode_mimeheader' => ['string', 'string'=>'string'],
-'mb_decode_numericentity' => ['string', 'string'=>'string', 'convmap'=>'array', 'encoding='=>'string', 'is_hex='=>'bool'],
+'mb_decode_numericentity' => ['string|false', 'string'=>'string', 'convmap'=>'array', 'encoding='=>'string', 'is_hex='=>'bool'],
 'mb_detect_encoding' => ['string|false', 'str'=>'string', 'encoding_list='=>'mixed', 'strict='=>'bool'],
 'mb_detect_order' => ['bool|array', 'encoding_list='=>'mixed'],
-'mb_encode_mimeheader' => ['string', 'str'=>'string', 'charset='=>'string', 'transfer_encoding='=>'string', 'linefeed='=>'string', 'indent='=>'int'],
-'mb_encode_numericentity' => ['string', 'string'=>'string', 'convmap'=>'array', 'encoding'=>'string', 'is_hex='=>'bool'],
+'mb_encode_mimeheader' => ['string|false', 'str'=>'string', 'charset='=>'string', 'transfer_encoding='=>'string', 'linefeed='=>'string', 'indent='=>'int'],
+'mb_encode_numericentity' => ['string|false', 'string'=>'string', 'convmap'=>'array', 'encoding'=>'string', 'is_hex='=>'bool'],
```

Comment:
I missed that encoding is optional - https://www.php.net/manual/en/function.mb-encode-numericentity.php